### DISCUSSION TITLE
Fastlane's fast file has deprecated 'default_language'

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#224] [Ideas]
I'm having all kinds of trouble getting Fastlane setup for the Skip HelloWorld deploy.  One of the glaring errors seems to be the
generated Darwin/fastlane/Fastfile contains a line for:

**default_language** has been removed (https://github.com/fastlane/fastlane/blob/master/deliver/MigrationGuide.md) I don't know the reason or the effects - but it gives errors in Skip's Fastlane processing.

**Comment by davidakoontz** [comment]
I noted the line is # commented out in this example:

<img width="476" alt="Screenshot 2024-09-08 at 12 02 51 PM" src="https://github.com/user-attachments/assets/7970b302-0ebb-48b8-83a5-86c38b96b674">

**Comment by marcprux** [comment]
Thanks for the heads-up! We will remove the `default_language` key from the `skip init` command in the next release of Skip.
